tests/example-apps/healthcheck-app/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<HTML>
+  <HEAD>
+    <meta http-equiv="Cache-Control" content="no-store" />
```

Comment:
I think we need to change this to an HTTP header instead. Only the browser will inspect the document contents, so it won't be honoured by any intermediaries like caching HTTP proxies or CDNs.

It might also be safer to use a combination of `max-age=0` and `private` because not all intermediary caches respect `no-store` or `no-cache` as you'd expect. For reference this is what GOV.UK does:
```
➜  paas-cf git:(master) curlh https://www.gov.uk/__canary__ | grep Cache-Control
Cache-Control: max-age=0,no-store,no-cache
```

PS: If you're really interested there is: https://github.com/alphagov/cdn-acceptance-tests